build-scripts/build-rpm.sh

```diff
@@ -4,7 +4,9 @@
 source $(dirname "$(readlink -f "$0")")/build-srpm.sh
 
 # Install build dependencies
-dnf builddep -y rpmbuild/SRPMS/*src.rpm
+if [ "${SKIP_BUILDDEP}" != "yes" ]; then
+    dnf builddep -y rpmbuild/SRPMS/*src.rpm
```

Comment:
Would it make sense to add sudo?

```  
    if [ "${SKIP_BUILDDEP}" != "yes" ]; then
          sudo dnf builddep -y rpmbuild/SRPMS/*src.rpm  
```

Reply:
Not sure you can run `sudo` on COPR

Reply:
@lsm5 PTAL

Reply:
Additional files of interest in podman: https://github.com/containers/podman/blob/main/podman.spec.rpkg and https://github.com/containers/podman/blob/main/.packit.sh

Reply:
@mwperina created the packaging for this repo. So, the question is for him. I only wanted to be able to use the scripts locally without `sudo`.
In any case, I think this is out of the scope of this PR. If the repo switches to `packit` it should be in a separate PR.

Reply:
@lsm5 At the moment we are building RPMs as a part of our github CI per PR, so RPMs are attached as artifacts of the relevant github job.

Also when the PR is merged we have added integration with COPR using `make srpm` and our nightly RPMs are published in https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/

So at the moment I think we have everything in place, but of course if needed we can reconsider using packit in future ...